cmd/init.go

```diff
@@ -45,6 +52,8 @@ func (cli *CLI) init(opts Options) int {
 
 				_, err := plugin.FindPluginPath(installCfg)
 				if os.IsNotExist(err) {
+					installed = true
+					fmt.Fprint(cli.outStream, builder.String())
```

Comment:
Printing working directory here can result in verbose output for each plugin. e.g.

**before**
```
====================================================
working directory : subdir

Installing "aws" plugin...
Installed "aws" (source: github.com/terraform-linters/tflint-ruleset-aws, version: 0.34.0)
Installing "google" plugin...
Installed "google" (source: github.com/terraform-linters/tflint-ruleset-google, version: 0.30.0)
```

**after**
```
====================================================
working directory : subdir

Installing "aws" plugin...
Installed "aws" (source: github.com/terraform-linters/tflint-ruleset-aws, version: 0.34.0)
====================================================
working directory : subdir

Installing "google" plugin...
Installed "google" (source: github.com/terraform-linters/tflint-ruleset-google, version: 0.30.0)
```

Reply:
If this change helps to minimize installation logging, we may be able to adopt simpler output.

```
Installing "aws" plugin in subdir...
Installed "aws" (source: github.com/terraform-linters/tflint-ruleset-aws, version: 0.34.0)
Installing "google" plugin in subdir...
Installed "google" (source: github.com/terraform-linters/tflint-ruleset-google, version: 0.30.0)
```

Reply:
True, I was just focused on minimizing output when no plugins are installed but we could extend this to have minimal output for all recursive calls. I am OK with how it is but I can make that change if you prefer.

Reply:
Yeah, it's OK if you do it or not. However, please fix https://github.com/terraform-linters/tflint/pull/2150#discussion_r1827019301.
I just thought it might be easier to fix this issue by changing the output.

Reply:
Ah thanks, I missed that. I've updated it to handle non-recursive calls correctly.